debugclient/getopt.c

```diff
@@ -0,0 +1,1258 @@
+/* Getopt for GNU.
+   NOTE: getopt is now part of the C library, so if you don't know what
+   "Keep this file name-space clean" means, talk to drepper@gnu.org
+   before changing it!
```

Comment:
I'm not at all knowlable in this licensing stuff, but was aware that this could be a problem. I can try to find something else. What would be OK and what is not?